packages/Tabs/src/components/Panel/Panel.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import panelStyles from "./Panel.styles";
+
+const propTypes = {
+  children: PropTypes.node,
+  isSelected: PropTypes.bool,
+};
+
+const defaultProps = {
+  children: null,
+  isSelected: false,
+};
+
+const Panel = props => {
+  const { isSelected, children, ...moreProps } = props;
+
+  if (!isSelected) {
+    return null;
+  }
+
+  return (
+    <div {...moreProps} css={panelStyles} role="tabpanel">
```

Comment:
@alexzherdev @nahumzs @oscarkwan 

I also don't really understand all that thing with `{...moreProps}`. For some components we use them, for the other - not. I don't see the pattern. 

And I feel kinda iffy about this approach. Isn't it making the component API more wobbly?

Reply:
I mean... we're supposed to use it wherever it makes sense for the consumer to put additional props on the element? If we're not doing it somewhere, that's probably because we haven't found that need yet.